packages/nimble-components/src/rich-text/editor/styles.ts

```diff
@@ -80,6 +80,15 @@ export const styles = css`
         border: ${borderWidth} solid rgba(${borderRgbPartialColor}, 0.1);
     }
 
+    ${/* Shared styles for all mention views at edit time. */ ''}
```

Comment:
There is one selector in the ProseMirror inserted content section below that looks liek:

```
li > p {
        margin-block: 0;
    }
```

And should be updated to:
```
.ProseMirror li > p {
        margin-block: 0;
    }
```

Reply:
Thanks for pointing it out. I have updated the same.